package.json

```diff
@@ -7,7 +7,7 @@
     "test": "forge test -v",
     "prettier:check": "prettier --check \"(contracts|test|script)/**/*.sol\"",
     "prettier:fix": "prettier --write \"(contracts|test|script)/**/*.sol\"",
-    "lint": "solhint 'contracts/**/*.sol' 'test/**/*.sol' 'script/**/*.sol'",
+    "lint": "solhint 'contracts/**/*.sol' 'script/**/*.sol'",
```

Comment:
```suggestion
    "lint": "solhint -w 3 'contracts/**/*.sol' 'script/**/*.sol'",
```

Reply:
Unfortunately we are not ready for this change yet.   Hopefully very soon!  And we have made strides!